### PR TITLE
History and single video page

### PR DESCRIPTION
**Pull request by shristabaruah**
- History page added
- single page video
- video is added to history onclick of a video in single page video

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -3,6 +3,10 @@
   --color-secondary: #051525a2;
   --color-border-hover: #22b2ee;
   --color-blue: #052b77;
+  --color-blue-grey:#5072A7;
+  --color-indigo:#3F00FF;
+  --color-morning-blue:#6050DC;
+  --color-royal-blue:#4169E1;
```
Comment (gadgetfather) [inline src/App.css:9]: Good variable naming for colors !

File: src/Services/getSingleVideo.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const getSingleVideo = async (setSingleVideo, videoId) => {
+  try {
+    const response = await axios.get(`/api/video/${videoId}`);
+    if (response.status === 200) {
+      setSingleVideo(response.data.video);
+    }
+  } catch (error) {
+    console.error(error);
```
Comment (gadgetfather) [inline src/Services/getSingleVideo.js:10]: You can use "hotToast" or "toastify" to show errors so user can know what is the issue

File: src/components/HorizontalCard/horizontal-card.jsx [hunk 3]
```diff
@@ -1,34 +1,54 @@
...
+    switch (location.pathname) {
+      case "/like":
+        removeFromLikesHandler(_id, token, likeDispatch);
+        break;
+      case "/history":
+        removeHistory(_id, token, historyDispatch);
+        break;
+      default:
+        break;
+    }
```
Comment (gadgetfather) [inline src/components/HorizontalCard/horizontal-card.jsx:32]: Good use of switchcase to make function dynamic.

**Comment by gadgetfather** [comment]
Functionality is working as intended. You can autoplay the video when someone goes to a single video page. 
Great work @shristabaruah

**Review by gadgetfather — APPROVED** [review]

**Review by NamrataRaikwar2002 — APPROVED** [review]
- UI looks nice
- functionality working properly
- Clean code
Great work

**Pull request merged**